### DISCUSSION TITLE
My other account got suspended because it was hacked

### DISCUSSION THREAD

**Discussion by Makkan-Innovations** [community/community#130634] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, my other github account : [](https://github.com/SegunCodes) got suspended this morning. After trying to see what caused it, apparently my other account was hacked yesterday and someone pushed a malicious code which resulted in the account been suspended. Is there something that can be done for my other account to be reinstated and the malicious code/repository deleted?

**Comment by LiteBrite82** [comment]
Hi @Makkan-Innovations 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
